lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -105,6 +108,13 @@ def rbs_inline_annotation?(line, comment)
             false
           end
         end
+
+        def steep_annotation?(line, comment)
+          return false unless line.match?(METHOD_DEFINITION)
+          return false unless comment.text.include?('steep')
```

Comment:
It seems that specs still pass even after removing these conditions. Can you either add specs or remove the unnecessary code if it is not needed?

Reply:
@koic 

Oops, I removed this condition since it is not needed.
It will be checked by a regular expression in `STEEP_REGEXP`.
Thanks for the review!!

Reply:
I removed this condition and I added test cases for some use cases.

In my project, `# steep:ignore` inline comment is written after method definition, so I defined this condition. However, I realized that it can be written after begin, end, class and module definition.